src/playbooks/with_build/vstat_health.yml

```diff
@@ -18,3 +18,14 @@
   roles:
     - vstat-vrs-health
 
+- hosts: vstats
```

Comment:
I think this needs to be:

hosts: vstats,primary_vstats,data_vstats

due to the way the hosts file keeps the vstats in separate groups.

Reply:
I think primary_vstats in already counted in the vstats. I will add data_vstats to the list